### PR TITLE
[i18n-ar] Translated file : `docs/source/ar/accelerate.md` into Arabic

### PR DESCRIPTION
**Pull request by AhmedAlmaghz**
## What does this PR do?
Translated the `docs/source/ar/accelerate.md` file of the documentation to `Arabic العربية`. I use transformers for my side-project and wanted to contribute. As a beginner, this seemed appropriate.

### Some questions:

- I can't access the pr documentation endpoint for `Arabic العربية`. Is there a missing step?
- I left all the other files alone after copying the English docs. Is this acceptable for future updates or should I add contents to the `_toctree.yml` only when translation is complete? Should I delete all other files for a cleaner review?
- In `Arabic`, people use both `Transformer` and `المحولات` to describe the model. However, as this is more of a product of HuggingFace I opted not to translate it. May you please let me know which you would prefer?

Thank you in advance for your review.

Part of #32435

### Before submitting

- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests), Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/) ?Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

 ### Who can review?
 @stevhliu , @MKhalusova , @abodacs, may you please review this PR?

### CODE REVIEW

**Comment by stevhliu** [comment]
Hi, when opening PRs, it is generally good not to open a ton at once. This gives reviewers time to review them without being overwhelmed. I recommend opening one or just a few PRs at once (you can even bundle them into one), waiting for them to be reviewed and merged, and then opening the next set of PRs.

**Comment by AhmedAlmaghz** [comment]
Thank you very much for your valuable comments - @stevhliu 
I took @abodacs comments and modified the previous pull requests to make the translation better.

File: docs/source/ar/accelerate.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## الإعداد
+
+ابدأ بتثبيت 🤗 Accelerate:
+
+```bash
+pip install accelerate
+```
+
+ثم قم باستيراد وإنشاء كائن [`~accelerate.Accelerator`]. سيقوم [`~accelerate.Accelerator`] تلقائيًا بالكشف عن نوع الإعداد الموزع الخاص بك وتهيئة جميع المكونات اللازمة للتدريب. لا تحتاج إلى وضع نموذجك على جهاز بشكل صريح.
```
Comment (abodacs) [inline docs/source/ar/accelerate.md:11]:
```suggestion
ثم قم باستيراد وإنشاء كائن [`~accelerate.Accelerator`]. سيقوم [`~accelerate.Accelerator`] تلقائيًا باكتشاف نوع الإعداد الموزع الخاص بك وتهيئة جميع المكونات اللازمة للتدريب. لن تحتاج إلى وضع نموذجك على جهاز بشكل معين.
```

File: docs/source/ar/accelerate.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+```py
+>>> from accelerate import Accelerator
+
+>>> accelerator = Accelerator()
+```
+
+## الاستعداد للتسريع
+
+الخطوة التالية هي تمرير جميع كائنات التدريب ذات الصلة إلى طريقة [`~accelerate.Accelerator.prepare`]. ويشمل ذلك DataLoaders للتدريب والتقييم، ونموذجًا ومُحَسِّنًا:
```
Comment (abodacs) [inline docs/source/ar/accelerate.md:21]:
```suggestion
الخطوة التالية هي تمرير جميع كائنات التدريب ذات الصلة إلى دالة الإعداد [`~accelerate.Accelerator.prepare`]. ويشمل ذلك DataLoaders للتدريب والتقييم، ونموذجًا ومُحَسِّنً المعاملات (optimizer):
```

File: docs/source/ar/accelerate.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+```py
+>>> train_dataloader, eval_dataloader, model, optimizer = accelerator.prepare(
+...     train_dataloader, eval_dataloader, model, optimizer
+... )
+```
+
+## الخلفي
+
+الإضافة الأخيرة هي استبدال `loss.backward()` النموذجي في حلقة التدريب الخاصة بك بأسلوب [`~accelerate.Accelerator.backward`] في 🤗 Accelerate:
```
Comment (abodacs) [inline docs/source/ar/accelerate.md:31]:
```suggestion
الإضافة الأخيرة هي استبدال الدالة المعتادة `loss.backward()` في حلقة التدريب الخاصة بك بدالة [`~accelerate.Accelerator.backward`] في 🤗 Accelerate:
```

File: docs/source/ar/accelerate.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+الخطوة التالية هي تمرير جميع كائنات التدريب ذات الصلة إلى طريقة [`~accelerate.Accelerator.prepare`]. ويشمل ذلك DataLoaders للتدريب والتقييم، ونموذجًا ومُحَسِّنًا:
+
+```py
+>>> train_dataloader, eval_dataloader, model, optimizer = accelerator.prepare(
+...     train_dataloader, eval_dataloader, model, optimizer
+... )
+```
+
+## الخلفي
```
Comment (abodacs) [inline docs/source/ar/accelerate.md:29]:
```suggestion
## الخلفي Backward
```

File: docs/source/ar/accelerate.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+ثم قم بتشغيل التدريب الخاص بك باستخدام:
+
+```bash
+accelerate launch train.py
+```
+
+### التدريب باستخدام دفتر ملاحظات
+
+يمكن أيضًا تشغيل 🤗 Accelerate في دفتر ملاحظات إذا كنت تخطط لاستخدام وحدات معالجة الرسوميات (TPUs) في Colaboratory. قم بتغليف كل الكود المسؤول عن التدريب في دالة، ومررها إلى [`~accelerate.notebook_launcher`]:
```
Comment (abodacs) [inline docs/source/ar/accelerate.md:109]:
```suggestion
يمكن أيضًا تشغيل 🤗 Accelerate في دفاتر إذا كنت تخطط لاستخدام وحدات معالجة الرسوميات (TPUs) في Colaboratory. قم بتغليف كل الكود المسؤول عن التدريب في دالة، ومررها إلى [`~accelerate.notebook_launcher`]:
```

**Review by abodacs — CHANGES_REQUESTED** [review]
السﻻم عليكم
@AhmedAlmaghz 
جزيل الشكر
تمت المراجعة

**Comment by AhmedAlmaghz** [comment]
أشكرك من اعماق قلبي أخي/ عبدالله
الله يبارك فيك 
@abodacs

**Review by abodacs — APPROVED** [review]

**Comment by abodacs** [comment]
@AhmedAlmaghz 
جزاك الله خيرا

ممكن تجول الحالة من draft to open

**Pull request closed without merging**